scripts/run.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+import sys
+
+# pylint: disable=wrong-import-position
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from allennlp.run import train_model_from_file
+from allennlp.common.checks import ensure_pythonhashseed_set
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def main():
+    usage = 'USAGE: train.py [param_file] [train]'
```

Comment:
Yeah - this looks a bit weird because there isn't a way to run only evaluation yet, but eventually I was thinking of there also being an `eval` option - this is what we previously had in `deep_qa` and probably analogous to what you are suggesting above.